Tophat.xcodeproj/project.pbxproj

```diff
@@ -855,7 +861,7 @@
 				);
 				LOCALIZATION_PREFERS_STRING_CATALOGS = YES;
 				MACOSX_DEPLOYMENT_TARGET = 15.0;
-				MARKETING_VERSION = 1.0;
+				MARKETING_VERSION = 1.11.1;
```

Comment:
Drive-by fixes to the versions of the extensions, they need to be in sync with the parent bundle.